lib/Settings/Personal.php

```diff
@@ -32,6 +32,8 @@ class Personal implements ISettings {
 
 	/** @var IConfig */
 	private $config;
+	/** @var \IURLGenerator */
```

Comment:
```suggestion
	/** @var IURLGenerator */
```
The interface is already in the namespace